lnbits/core/views/admin_api.py

```diff
@@ -54,13 +55,16 @@ async def api_get_settings(
     return admin_settings
 
 
+class UpdateSettings(EditableSettings):
```

Comment:
- is this needed now, extra fields will be ignored, right?

Reply:
its intentionally, so that if you try to send invalid settings it will error instead of ignoring. I didn't want that for `EditableSettings` though because it can be annoying in development when you have some settings from another branch in your db and now you get errors on startup because they dont exist in current branch

Reply:
ok, this class should declared be next to `EditableSettings` (not in the api file)